tests/nolibc/main.c

```diff
@@ -0,0 +1,40 @@
+#include <hw/kernel.h>
+
+#include <bmk-core/errno.h>
+#include <bmk-core/printf.h>
+#include <bmk-core/string.h>
+
+#include <rump/rump.h>
+
+#include "nolibc.h"
+
+static ssize_t
+writestr(int fd, const char *str)
+{
+	return rump_sys_write(fd, str, bmk_strlen(str));
+}
+
+void
+mainthread(void *cmdline)
+{
+	int rv, fd;
+
+	rv = rump_init();
+	bmk_printf("rump kernel init complete, rv %d\n", rv);
+
+	writestr(1, "Hello, stdout!\n");
+
+	bmk_printf("open(/notexisting): ");
+	fd = rump_sys_open("/notexisting", 0);
+	if (fd == -1) {
+		if (errno == BMK_ENOENT) {
```

Comment:
This test should be against `RUMP_ENOENT`. Yes, it happens to have the same value as `BMK_ENOENT`, but anway.